code/modules/mob/mob_grab_specials.dm

```diff
@@ -120,7 +120,7 @@
 	attacker.next_move = world.time + 20 //2 seconds, also should prevent user from triggering this repeatedly
 	if(do_after(attacker, 20, progress=0) && target)
 		visible_message(SPAN_DANGER("...And falls backwards, slamming the opponent back onto the floor!"))
-		var/damage = ((attacker.stats.getStat(STAT_ROB) / 2) + 15)
+		var/damage = (attacker.stats.getStat(STAT_ROB) + 15)
```

Comment:
![image](https://user-images.githubusercontent.com/65828539/151430900-b4def1c8-5940-4b95-8182-1fbdf72da5cd.png)

This need a cap.

```suggestion
		var/damage = min(50, attacker.stats.getStat(STAT_ROB) + 15)
```

Reply:
Maximum 65 damage? **From a red grab, where you can almost instakill a person by using a knife on them**? No.

Reply:
No, maximum 50 damage. Using a knife is more effective than bare hands, have sense.

Reply:
UNLESS you are ripped as fuck. Who cares about realism? Knife is also faster and you don't fall.

Reply:
> Who cares about realism?

Classic argument again, justifies just about anything.

Reply:
Ignoring the inane babbling about realism, he's right that you should put a cap on this. This also knocks the suplex victim down for longer than the suplexer, so it gives some time to kick them while down. I don't think it really matters much if you pick 50 or 65. If you have gripes with knife, I advise fixing that instead of trying to compare with this.

Reply:
OR...

We could leave it uncapped for this weekend for shits and giggles.

Reply:
> OR...
> 
> We could leave it uncapped for this weekend for shits and giggles.

Insanely based tbh, but unwise.

Reply:
Unlike a knife to the throat, to my knowledge this does not bypass armor. Should be considered.

Reply:
After I've talked to the voices in my head, I've decided to cap the damage at 65. Will push when I get home.